### DISCUSSION TITLE
Notification after request has finished processing?

### DISCUSSION THREAD

**Discussion by ThijmenDam** [seerr-team/seerr#309] [Q&A] [answered]
Hi, I'd like send a notification to my phone whenever Jellyseerr has finished processing a request, but I am unsure if this is possible. 

I have managed to send a notification on "Request Available", but this is only triggered after a Radarr/Sonarr scan, and not after the request has finished processing. This is a bit of a problem since a Radarr/Sonarr scan is performed at max once per hour, and I'd like to receive a notification as soon as the media is on my disk.

**Comment by fallenbagel** [comment]
Availability is determined by Availability in jellyfin (main functionality) and if its not showing Availability through jellyfin and only by *arr then that means your metadata is missing the required IDs.

How jellyseerr works is, it does a recently added jellyfin scan every 5 mins indefinitely which will makr if a series or movie is available on jellyfin or not and also add rhe play on jellyfin button. A media to be recently added is determined by its date added metadata which is determined by jellyfin itself through the date modified.

To debug the issue why its not showing Availability right after scanning recently added, first disable enable scan on all *arr. Then request and once a media is added manually run the jellyfin recently added scan. If it does not pick up then run full scan. If it does pick up, it's the date modified issue (which can be fixed). However, if it still does not pick up and only picks up Availability through the *arr stack only, then that means your metadata is lacking the required tmdb IDs. The solution for this issue requires a few steps (rather easy fix) but could you try the previous debug methods first?

Notification is sent after the recently added scan determines that the item is on jellyfin every 5 mins. So from your description I'm assuming that this scan is not picking up your recently added media and thats the reason why you're not getting the notification immediately (it is supposed to as the that scan is run very frequently and is also the core functionality and how the notification system is supposed to work).

**Comment by ThijmenDam** [comment, marked as answer]
Thanks for your extensive answer @Fallenbagel. My metadata turned out to be okay, though I figured the notifications would only be sent on a full Radarr/Sonarr scan - which is not the case. I totally missed that upon a "Jellyfin Recently Added Scan" the notification is sent too if new media is detected. This scan is performed automatically each 5 minutes, so that is very reasonable.
